electron/main.js

```diff
@@ -108,6 +100,21 @@ if (process.platform === 'win32') {
   ipcMain.on('wrapper-restart', function() {
     squirrel.installUpdate();
   });
+
+  if (shouldQuit) {
+    // Stop further execution on update to prevent second tray icon
+    return;
+  }
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+if (process.platform === 'linux') {
```

Comment:
What is also a nice coding pattern is to use functions over conditions. 

**Example:**

```javascript
function isUnity() {
  if (process.env.XDG_CURRENT_DESKTOP && process.env.XDG_CURRENT_DESKTOP.includes('Unity')) {
    return true;
  }
  return false;
}

if (isUnity()) {
  process.env.XDG_CURRENT_DESKTOP = 'Unity';
}
```